Description: Drop async-std dep since it's not in Debian
Author: John Goerzen <jgoerzen@complete.org>
Last-Update: 2022-10-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,11 +27,6 @@
 license = "Apache-2.0 WITH LLVM-exception OR Apache-2.0 OR MIT"
 repository = "https://github.com/sunfishcode/io-lifetimes"
 
-[dependencies.async-std]
-version = "1.9.0"
-features = ["unstable"]
-optional = true
-
 [dependencies.fs-err]
 version = "2.6.0"
 optional = true
